molecule/default/tests/test_verify_mbu_installed.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pytest
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('os-infra_hosts')[:1]
+
+
+backup_folder_on_host = '/openstack/backup'
+backup_folder_on_container = '/var/backup'
+
+
+@pytest.mark.incremental
+class TestVerifyMBUInstall(object):
+    @pytest.mark.test_id('f0ef4020-a16f-11e8-97f3-fe0851898bb5')
+    @pytest.mark.jira('asc-262')
+    def test_verify_backup_folder_on_host(self, host):
+        assert host.file(backup_folder_on_host).exists
+
+    @pytest.mark.test_id('4a7ed621-a1a3-11e8-8948-6a0003552100')
+    @pytest.mark.jira('asc-262')
+    def test_veiry_backup_folder_on_container(self, host):
+        cmd1 = "ls {}".format(backup_folder_on_host)
+        output = host.run(cmd1).stdout
+        containers = output.split('\n')
```

Comment:
You could use `stdout_lines` rather than splitting `stdout`.

Reply:
using `stdout_lines` will throw error like ` E       AttributeError: 'unicode' object has no attribute 'stdout_lines'`, probably it's because the nature of `host.run` output from Molecule